README.md

```diff
@@ -8,44 +8,57 @@ This repository contains the source code for the automation tests for the Salesf
+│   └── extensions                          # Directory containing the salesforce extensions
+│       ├── salesforcedx-vscode-core-59.0.0.vsix
+│       ├── salesforcedx-vscode-apex-59.0.0.vsix
+│       └── ...
+└── ...
+```
 
-For the VSCode extension (`{location}/salesforcedx-vscode`), you will need to run `npm install` but you do not need to compile or perform any other steps.
-
-After the dependencies have been installed, open the folder in Visual Studio Code, then debug using the `Debug All Automation Tests` configuration (or run using the `Run All Automation Tests` configuration).
+To install the test dependencies, run `npm install`. You do not need to compile - when running the e2e automation tests, the code is dynamically compiled.
 
 ### Environment Variables
 
 The following is a list of environment variables that are used with this project. Each has a default value and are obtained using the [environmentSettings](test/environmentSettings.ts) class.
 
-#### DEV_HUB_ALIAS_NAME
-
-Default value: `vscodeOrg`
+- DEV_HUB_ALIAS_NAME
 
-#### DEV_HUB_USER_NAME
+  - Default value: `vscodeOrg`
 
-Default value: `svcideebot@salesforce.com`
+- DEV_HUB_USER_NAME
 
-#### EXTENSION_PATH
+  - Default value: `svcideebot@salesforce.com`
 
-Default value: `{cwd}/../../salesforcedx-vscode/packages`
+- EXTENSION_PATH
 
-The default folder structure is `{location}/salesforcedx-vscode` and `{location}/salesforcedx-vscode-automation-tests`, and if both repos are at the same location no changes are needed for `EXTENSION_PATH`. If your folder structure does not match this, EXTENSION_PATH will need to be set to the relative path to 'salesforcedx-vscode/packages' .
+  - Default value: `{cwd}/../../salesforcedx-vscode/extensions`
 
-#### THROTTLE_FACTOR
+    Note: If your folder structure does not match the folder structure shown above in Getting Started section, `EXTENSION_PATH` will need to be set to the relative path to 'salesforcedx-vscode/extensions'. If it does match, then no changes are needed for.
 
-Default value: 1
+- THROTTLE_FACTOR
+  - Default value: `1`
 
 ### Dev Hub
 
-A requirement of this project is for a dev hub to have been enabled on the user's machine. The default dev hub name is "vscodeOrg" and the default username is "svcideebot@salesforce.com", though this can be configured with the `DEV_HUB_ALIAS_NAME` and `DEV_HUB_USER_NAME` environment variables.
-Run Task : Authorize DevHub - E2E Testing Org through command palette (Cmd+shft+P).
-Once you are connected to the org with `DEV_HUB_ALIAS_NAME` and `DEV_HUB_USER_NAME` you can run single or all end-to-end test suites.
+A requirement of this project is for a Dev Hub to have been enabled on the user's machine. The default Dev Hub name is "vscodeOrg" and the default username is "svcideebot@salesforce.com", though this can be configured with the `DEV_HUB_ALIAS_NAME` and `DEV_HUB_USER_NAME` environment variables.
+Run Task: Authorize DevHub - E2E Testing Org through command palette (Cmd+shft+P).
```

Comment:
```suggestion
Run Task: Authorize DevHub - E2E Testing Org through command palette (Cmd+shift+P).
```

Reply:
FYI - I ran this command and it used `sfdx auth web login`, which is an SFDX-style command that is no longer supported.  Not a change that needs to be done in this PR but just some info.
![Screenshot 2023-12-19 at 9 53 56 PM](https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/139700604/6f9d7062-f5ca-46dd-b085-c571e96ab700)